src/main/java/ua/lviv/iot/novaPoshtaAPI/datastorage/dal/CourierFileStore.java

```diff
@@ -0,0 +1,181 @@
+package ua.lviv.iot.novaPoshtaAPI.datastorage.dal;
+
+import org.springframework.stereotype.Component;
+import ua.lviv.iot.novaPoshtaAPI.Util;
+import ua.lviv.iot.novaPoshtaAPI.model.Courier;
+
+import java.io.File;
+import java.io.OutputStreamWriter;
+import java.io.FileOutputStream;
+import java.io.Writer;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.time.LocalDate;
+import java.util.*;
+import java.util.stream.Collectors;
+
+@Component
+public class CourierFileStore {
```

Comment:
ok, can't see where is a HashMap to cache the data from file, not to read it from file each time